src/ActionMonitor/Monitors/NavMenuMonitor.php

```diff
@@ -150,7 +150,7 @@ public function log_diffed_menus( array $old_locations, array $new_locations ) {
 				$this->log_action(
 					[
 						'action_type'         => 'DELETE',
-						'title'               => $removed_menu_id,
+						'title'               => 'Menu: ' . $removed_menu_id,
```

Comment:
This should be the menu title or just the menu id (`#${removed_menu_id)`)

![Uploading Screen Shot 2020-12-14 at 5.42.47 PM.png…]()